widgetastic_manageiq.py

```diff
@@ -875,6 +876,27 @@ def first_page(self):
     def last_page(self):
         self.paginator.last_page()
 
+    def pages(self):
+        """Generator to iterate over pages, yielding after moving to the next page"""
+        if self.exists:
+            # start iterating at the first page
+            if self.cur_page != 1:
+                logger.debug('Resetting paginator to first page')
+                self.first_page()
+
+            # Adding 1 to pages_amount to include the last page in loop
+            for page in range(1, self.pages_amount + 1):
```

Comment:
Not quite sure what you mean here - I still need to add 1 to `pages_amount` to get the correct behavior for the last page, do you mean drop the 'page' assignment like:

```python
for _ in (self.pages_amount + 1):
```